jobs/build-node-image.Jenkinsfile

```diff
@@ -109,16 +109,41 @@ lock(resource: "build-node-image") {
                      label_args = ["--label", "coreos.build.manifest-list-tag=${unique_tag}"]
                  }
 
+                // pull the base RHCOS image and get it's digest and build ID
+                def base_image_manifest = readJSON(text: shwrapCapture("""
+                                                    skopeo inspect --authfile \$REGISTRY_AUTH_FILE --no-tags \
+                                                    docker://$build_from
+                                                    """))
+                def base_image_digest = base_image_manifest["Digest"]
+                def base_build_id = base_image_manifest["Labels"]["org.opencontainers.image.version"]
+
+                build_from = build_from.split(':')[0]
```

Comment:
It should be part of `build_and_push_image/build_remote_image`, the idea is to keep as much as possible in the libs. Let's create a new function to get it.